packages/report-server/src/reportBuilder/functions/basic.ts

```diff
@@ -40,3 +40,7 @@ export const exists = (val: any): boolean => {
 export const notExists = (val: FieldValue): boolean => {
   return val === undefined;
 };
+
+export const length = (val: FieldValue[]): number => {
```

Comment:
```suggestion
export const length = (val: any[]): number => {
```
Being nitpicky, but we're passing in a `Row[]` when we call `length($table)` 👍